### ISSUE TITLE
errors when receiving data from apple pipeline

### ISSUE THREAD

**Issue by woolfg** [openpodcast/api#119]
```
api-api-1  | Error: Not authorized: Token format is invalid
api-api-1  |     at AuthController.<anonymous> (/app/dist/auth/AuthController.js:45:29)
api-api-1  |     at Generator.next (<anonymous>)
api-api-1  |     at /app/dist/auth/AuthController.js:8:71
api-api-1  |     at new Promise (<anonymous>)
api-api-1  |     at __awaiter (/app/dist/auth/AuthController.js:4:12)
api-api-1  |     at AuthController.handleRequest (/app/dist/auth/AuthController.js:38:50)
api-api-1  |     at /app/dist/utils/expressHelpers.js:12:20
api-api-1  |     at Layer.handle [as handle_request] (/app/node_modules/express/lib/router/layer.js:95:5)
api-api-1  |     at trim_prefix (/app/node_modules/express/lib/router/index.js:328:13)
api-api-1  |     at /app/node_modules/express/lib/router/index.js:286:9
api-api-1  | Error: Unknown endpoint in meta: showTrends/ListeningTimeEpisodes
api-api-1  |     at AppleConnector.<anonymous> (/app/dist/api/connectors/AppleConnector.js:116:23)
api-api-1  |     at Generator.next (<anonymous>)
api-api-1  |     at /app/dist/api/connectors/AppleConnector.js:8:71
api-api-1  |     at new Promise (<anonymous>)
api-api-1  |     at __awaiter (/app/dist/api/connectors/AppleConnector.js:4:12)
api-api-1  |     at AppleConnector.handleRequest (/app/dist/api/connectors/AppleConnector.js:28:16)
api-api-1  |     at ConnectorApi.<anonymous> (/app/dist/api/ConnectorApi.js:42:36)
api-api-1  |     at Generator.next (<anonymous>)
api-api-1  |     at /app/dist/api/ConnectorApi.js:8:71
api-api-1  |     at new Promise (<anonymous>)
api-api-1  | Error: must have required property 'podcastEpisodeITunesTitle' <> must have required property 'podcastEpisodeITunesTitle' <> must have required property 'podcastEpisodeITunesTitle' <> must have required property 'podcastEpisodeITunesTitle' <> must have required property 'podcastEpisodeITunesTitle' <> must have required property 'podcastEpisodeITunesTitle' <> must have required property 'podcastEpisodeITunesTitle' <> must have required property 'podcastEpisodeITunesTitle' <> must have required property 'podcastEpisodeITunesTitle' <> must have required property 'podcastEpisodeITunesTitle' <> must have required property 'podcastEpisodeITunesTitle' <> must have required property 'podcastEpisodeITunesTitle' <> must have required property 'podcastEpisodeITunesTitle' <> must have required property 'podcastEpisodeITunesTitle'
api-api-1  |     at validateJsonApiPayload (/app/dist/api/JsonPayloadValidator.js:17:15)
api-api-1  |     at AppleConnector.<anonymous> (/app/dist/api/connectors/AppleConnector.js:31:67)
api-api-1  |     at Generator.next (<anonymous>)
api-api-1  |     at /app/dist/api/connectors/AppleConnector.js:8:71
api-api-1  |     at new Promise (<anonymous>)
api-api-1  |     at __awaiter (/app/dist/api/connectors/AppleConnector.js:4:12)
api-api-1  |     at AppleConnector.handleRequest (/app/dist/api/connectors/AppleConnector.js:28:16)
api-api-1  |     at ConnectorApi.<anonymous> (/app/dist/api/ConnectorApi.js:42:36)
api-api-1  |     at Generator.next (<anonymous>)
api-api-1  |     at /app/dist/api/ConnectorApi.js:8:71
```

**Comment by woolfg** [comment]
- / => Error: Not authorized: Token format is invalid: create a blank / page or 404
- Error: must have required property 'podcastEpisodeITunesTitle' => problem as not present for anchor.fm podcasts, remove requires property

**Comment by woolfg** [comment]
/ => Error: Not authorized: Token format is invalid wil be fixed by #103

**Comment by woolfg** [comment]
known endpoint in meta: showTrends/ListeningTimeEpisodes => #121
